PHPCompatibilityMagento/ruleset.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<ruleset name="PHPCompatibilityMagento">
+    <description>Magento 2 specific ruleset which checks for PHP cross version compatibility.</description>
+    <rule ref="../vendor/phpcompatibility/php-compatibility/PHPCompatibility">
+        <exclude name="PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound" />
+        <exclude name="PHPCompatibility.FunctionUse.RequiredToOptionalFunctionParameters" />
```

Comment:
This sniff had to be excluded because they return a `PHP Fatal error:  Cannot declare class PHPCompatibility\Sniffs\FunctionUse\RequiredToOptionalFunctionParametersSniff, because the name is already in use` error. The same happens with the following 2 exclusions. Ideas are welcomed.

Reply:
@svera can you please check how these names are allocated initially.

Also, can you please add the compatibility rule to Magento2 standard ruleset.

Reply:
@sivaschenko it seems the fatal error mentioned above is a known issue

https://github.com/PHPCompatibility/PHPCompatibility/issues/793

unfortunately, the latest release was published before the fix of this issue. What do you think we should do here?

Reply:
@sivaschenko  I finally found the issue. PHPCompatibility was not registered in PHPCS, thus the only way to include it was using its path in the `vendor` folder which, in turn, caused a bug in PHPCompatibility to raise up. Solution is easy, just register PHPCompatibility as an installed path of PHPCS through `composer.json`